vroom/main.vroom

```diff
@@ -18,7 +18,30 @@ stub that out for vroom.
   :endfunction
   :call maktaba#test#Override('repeat#set', 'FakeRepeat')
 
-We'll also stub it out to not care whether certain executables are installed on
+
+Before we stub out availability checks, test :FormatCode for unavailable plugins
```

Comment:
Heh, I know how you feel. Sure, if you want to just address the other comment and submit, I can take care of it.